optimizely/marketing-website
# Pricing page buttons

### What this PR does
1. Addresses:  [MKT-2151](https://optimizely.atlassian.net/browse/MKT-2151) and [MKT-2092](https://optimizely.atlassian.net/browse/MKT-2092)
1. Add buttons to /plans so it appears like the experiment on /pricing
2. Adjust JS so dialogs work on all buttons on the page
3. Remove old pages /features-and-plans and /feature-list which were associated with an old experiment on the pricing page.
4. Fix bug where click-handler is not attached to button on foreign domains.
5. Remove requests to `/experiment/load_recent?max_experiments=5` as they are no longer used.
6. Adds more contact information per [MKT-2103](https://optimizely.atlassian.net/browse/MKT-2103)

### To Test
1. Visit staging and go to /plans
2. Click various "Test it out" buttons and verify they bring up the Create Account modal.
3. Navigate to /mobile and create an account. This will be an account without a plan. Navigate back to /plans and press "Test it out" button. The button should have a disabled class and the page should navigate to `https://app.optimizely.com/welcome`
4. Return to /plans and refresh. The "Test it out" button should be gone and "Your current plans" should be in its place.
